docs/docs/widgets/checkbox.md

```diff
@@ -1,129 +1,133 @@
 ---
-id: checkbox
-title: Checkbox
----
+
+id: checkbox  
+title: Checkbox  
+---  
 # Checkbox
 
 The **Checkbox** component allows users to make a binary choice, such as selecting or deselecting an option.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 ### Data
 
 | Property       | Description    | Expected Value         |
 |:---------------|:---------------|:-----------------------|
-| Label          | The text to be used as the label for the checkbox.          | String (e.g., `Select payment preference`).                 |
-| Default status | Sets the default status when the app is loaded.                | Toggle the on/off switch or click on `fx` and dynamically set the value. |
+| Label          | The text to be used as the label for the checkbox.          | String (e.g., **Select payment preference**).                 |
+| Default status | Sets the default status when the app is loaded.                | Toggle the on/off switch or click on **fx** and dynamically set the value. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
 | <div style={{ width:"100px"}}> Event </div>     | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| On change | On change event is triggered when checkbox input is changed. |
-| On check (deprecated) | On check event is triggered when checkbox input is checked. |
-| On uncheck (deprecated)| On uncheck event is triggered when checkbox input is unchecked. |
+| On change | The on change event is triggered when the checkbox input is changed. |
```

Comment:
You need to update the content of this table referring to the reference given to you in the issue, you don't need to exactly copy from the reference but you have to frame the sentence in a similar way.